sormas-ui/pom.xml

```diff
@@ -62,6 +62,13 @@
 		   <artifactId>extended-token-field</artifactId>
 		</dependency>
 
+		<!--ADDED VADDIN DEPENDENCY FOR LOADING WIDGET-->
```

Comment:
This is not needed. The widgetset is compiled based on the sormas-widgetset project